linkerd/app/inbound/src/lib.rs

```diff
@@ -268,10 +265,9 @@ impl Config {
                         .push(metrics.stack.layer(stack_labels("profile"))),
                 ),
             )
+            .spawn_buffer(buffer_capacity)
```

Comment:
i wasn't sure if we wanted to put an idle timeout here or not. i _think_ we only want that at the request level?

Reply:
i think that's right -- because the idle timeout is part of cache eviction. i'll give this a closer look though.